modules/core/arrow-core-data/src/test/kotlin/arrow/extensions/ComposedInstancesTest.kt

```diff
@@ -130,7 +106,6 @@ class ComposedInstancesTest : UnitSpec() {
 
     testLaws(
       FunctorLaws.laws(ComposedFunctor(Option.functor(), NonEmptyList.functor()), cf, EQ_OPTION_NEL),
-      FunctorFilterLaws.laws(ComposedFunctorFilter(OptionT.functorFilter(Id.monad()), OptionT.functorFilter(NonEmptyList.monad())), { OptionT.just(Id.monad(), OptionT.just(NonEmptyList.monad(), it)).nest() }, EQ_OPTIONT_ID_NEL),
```

Comment:
What happened here? Why was this deleted?

Reply:
It needed to move to mtl. We had to seperate the Transformer Tests from the Core one